reference/under_the_hood/storage.md

```diff
@@ -58,10 +58,16 @@ This dataset is a JSON file of 8.6 MB and has 19,553 documents. When we index th
 
 | Raw JSON | MeiliSearch database size on disk | RAM usage | Virtual memory usage |
 | -------- | --------------------------------- | -------------------- | ------------------- |
-| 9.1 MB   | 320 MB                            | ≃ 426 MB             | 205 Gb (memory map) |
+| 9.1 MB   | 224 MB                            | ≃ 305 MB             | 205 Gb (memory map) |
```

Comment:
Just to be sure, this is still using the `movies.json` dataset, right?

Reply:
Yes! Why?

Reply:
No reason, just wanted to confirm it =)